src/utils/Utils.ts

```diff
@@ -27,10 +26,36 @@ export function printMessage(message: string, title?: string, indent?: number) {
   print(message, console.log, title, indent);
 }
 
-export function printWarning(message: string, title?: string, indent?: number) {
+export function printWarning(
+  message: string,
+  indent?: number,
+  title = "Warning:"
+) {
   print(message, console.warn, title, indent);
 }
 
 export function printError(message: string, title?: string, indent?: number) {
   print(message, console.error, title, indent);
 }
+
+/**
+ * Returns the error as an AxiosError if it is one, otherwise it will _exit the program_.
```

Comment:
This does mean making a hard decision to kill the program any time an API call results in a failure that is not related to the status of the response. It may be completely fair to exit on these errors, as the may be things like memory issues or infinite loops, however, there may be something said for attempting to save what ever information might exist at the time (though this might be difficult).